user-impl/src/bundle/bundle.conf

```diff
@@ -0,0 +1,28 @@
+version = "1"
+name = "user"
+compatibilityVersion = "1"
+system = "user"
+systemVersion = "1"
+nrOfCpus = 0.1
+memory = 268435456
+diskSpace = 200000000
+roles = ["web"]
+components = {
+  user = {
+    description = "user"
+    file-system-type = "universal"
+    start-command = ["user/bin/user"]
+    endpoints = {
+      "user" = {
+        bind-protocol = "http"
+        bind-port = 0
+        services = ["http://:9000/user","http://:9000/api/user?preservePath"]
```

Comment:
I think this service URI causes an issue in ConductR. ConductR creates a mapping using the service name `user` which, in `online-auction-java` clashes with `web-gateway`'s `/user`. 

The change in `UserService` hasn't been pushed yet since it's still unclear wether ConductR should stop doing that. See https://github.com/lagom/online-auction-java/issues/32